theoplayer/getting-started/01-sdks/01-web/01-getting-started-accessibility.mdx

```diff
@@ -0,0 +1,63 @@
+
+## User Agent Accessibility Guidelines (UAAG)
+
+Under the WCAG is the User Agent Accessibility Guidelines (UAAG), which encompasses web browsers, browser extensions and media players, and the UAAG targets developers. The UAAG was created to primarily provide alternative information for those who want to remain compliant with WCAG, but are using a variety of different technologies and platforms. For example, some accessibility needs are met through the browser the viewer is using, while others are better met through the media content.
+
+UAAG is seen as relevant for the future of accessibility, especially for mobile device accessibility. Mobile devices show the need for accessibility within a combination of UI, the content and applications used. UAAG also highlights the need for more accessibility features that work seamlessly across devices, platforms, and browsers.
+If you would like to explore more about UAAG specifically, check out the documentation with [standards and guidelines from WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/).
+
+## Why is WCAG important for Video
+
+Complying with the WCAG is crucial for all publishers of online content, and particularly in video content. By ignoring accessibility, you are losing a large chunk of your audience. By a large chunk, we mean 15% of the world’s population who have at least one disability. By implementing accessibility features you are including everyone, reaching more viewers and increasing your traffic and SEO rankings.
+
+Not only is accessibility important for rankings and to those with disabilities, but it is also adds to the viewer experience and usability for non-disabled viewers as well. For example, if a viewer is in a noisy or quiet environment, or the viewer is a non-native speaker of the language used in the video, subtitles will allow them to still enjoy the video content. Keyboard accessibility can be of great use to those viewers who may not have a mouse or have a temporary injury.
+
+# Accessibility Features
+
+OptiView Player integrates the following features for accessibility:
+
+### Text Tracks
+
+Text Tracks (i.e. video captions or subtitles) are the audio from a video, but in text form. This includes words that are spoken, who is speaking (if it isn’t clear) and important noises such as music. This can help viewers who are hearing impaired in any capacity, or those with cognitive or learning disabilities. These captions can also benefit those general viewers whom, for example, may be in a loud, or silent, viewing environment. You can seamlessly integrate this feature into your OptiView player. For more information on integrating Text Tracks into OptiView player and the supported formats see our [guide on text tracks](/theoplayer/how-to-guides/texttrack/introduction/).
```

Comment:
question: @dcoffey3296 Should the 2 "player" occurrences here be capitalized?

nitpick: I'd put a comma after "supported formats".